sample/build.gradle

```diff
@@ -0,0 +1,57 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+
+
+    defaultConfig {
+        applicationId "com.urbanairship.sample"
+        minSdkVersion 16
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    repositories {
+        maven {
+            url "https://urbanairship.bintray.com/android"
+        }
+    }
+
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+
+    implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+    testImplementation 'junit:junit:4.12'
+    androidTestImplementation 'com.android.support.test:runner:1.0.2'
+    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
+
+    // Segment
+    implementation 'com.segment.analytics.android:analytics:4.+'
+
+    // Airship Integration
+    // apps add integration using the following dependency 
+//    implementation 'com.urbanairship.android:segment-integration:+'
+    //  only this sample app uses the following local dependency
+    implementation project(":segment-integration")  
+
+    implementation 'com.urbanairship.android:urbanairship-core:10.0.2'
```

Comment:
I don't understand why I had to add this dependency to access the core. Shouldn't I get this through the segment-integration dependency? I tried `api` also, but no luck.

Reply:
Did you try using api on the segment-integration build.gradle or just here? I think that's the place it would matter.

Reply:
Cleaned up.